### ISSUE TITLE
Support  list all lint by use cargo-clippy cli

### ISSUE THREAD

**Issue by baoyachi** [rust-lang/rust-clippy#8642]
### Description

Should be list all `rust-clippy` lint by used cli.

Like this:
```bash
~ cargo-clippy -list
absurd_extreme_comparisons
correctness deny 
almost_swapped
correctness deny 
approx_constant
correctness deny 
as_conversions
restriction allow 
assertions_on_constants
style warn 
assign_op_pattern
style warn 
...
...
...
```

### Version

```text
rustc 1.59.0 (9d1b2106e 2022-02-23)
binary: rustc
commit-hash: 9d1b2106e23b1abd32fce1f17267604a5102f57a
commit-date: 2022-02-23
host: x86_64-apple-darwin
release: 1.59.0
LLVM version: 13.0.0
```
